cmake/onnxruntime_external_ops.cmake

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+FIND_PACKAGE(PythonLibs)
+FIND_PACKAGE(NumPy)
+
+if(NOT PYTHON_INCLUDE_DIR)
+  set(PYTHON_NOT_FOUND false)
+  exec_program("${PYTHON_EXECUTABLE}"
+    ARGS "-c \"import distutils.sysconfig; print(distutils.sysconfig.get_python_inc())\""
+    OUTPUT_VARIABLE PYTHON_INCLUDE_DIR
+    RETURN_VALUE PYTHON_NOT_FOUND)
+  if(${PYTHON_NOT_FOUND})
+    message(FATAL_ERROR "Cannot get Python include directory. Is distutils installed?")
+  endif(${PYTHON_NOT_FOUND})
+endif(NOT PYTHON_INCLUDE_DIR)
+
+if(NOT NUMPY_INCLUDE_DIR)
+  set(NUMPY_NOT_FOUND false)
+  exec_program("${PYTHON_EXECUTABLE}"
+    ARGS "-c \"import numpy; print(numpy.get_include())\""
+    OUTPUT_VARIABLE NUMPY_INCLUDE_DIR
+    RETURN_VALUE NUMPY_NOT_FOUND)
+  if(${NUMPY_NOT_FOUND})
+    message(FATAL_ERROR "Cannot get NumPy include directory: Is NumPy installed?")
+  endif(${NUMPY_NOT_FOUND})
+endif(NOT NUMPY_INCLUDE_DIR)
+
+include_directories(${PYTHON_INCLUDE_DIR})
```

Comment:
You will find duplicated include header entry in your vc project.
PYTHON_INCLUDE_DIR will occur twice.

Reply:
![image](https://user-images.githubusercontent.com/48490400/59051616-2adf3100-8842-11e9-9781-287f7962cbac.png)

Reply:
Here is the build error without the including.